packages/component-library-vue/src/UtrechtSelect.vue

```diff
@@ -1,17 +1,28 @@
-<script setup lang="ts">
-  defineEmits<(_type: "update:modelValue", _value: unknown) => void>();
+<script lang="ts">
+  import { defineComponent } from "vue";
+  import { useModelWrapper } from "./helpers";
```

Comment:
Why not directly import from `./modelWrapper`? Seems like having two files is not necessary.

Reply:
I was thinking about making it nicer  when you import the helper's functions from one path, but I will change it to your suggestion because currently, we have one function